src/components/AddressDisplay/AddressDisplayTs.ts

```diff
@@ -45,13 +45,14 @@ export class AddressDisplayTs extends Vue {
      * Load transaction details
      * @return {Promise<void>}
      */
-    protected async loadDetails(): Promise<void> {
+    @Watch('address', { immediate: true })
+    async loadDetails(): Promise<void> {
         this.descriptor = '';
         if (!this.address) {
             return;
         }
         // in case of normal transfer, display pretty address
-        if (this.address instanceof Address) {
+        if (this.address && this.address instanceof Address) {
```

Comment:
Do we really need this change here? (pls check line 51, already checking if the address is truthy)